pallets/gear-builtin/src/lib.rs

```diff
@@ -93,6 +95,9 @@ pub enum BuiltinActorError {
     /// Actor's inner error encoded as a String.
     #[display(fmt = "Builtin execution resulted in error: {_0}")]
     Custom(LimitedStr<'static>),
+    /// Occurs if a builtin actor execution does not fit in the current block.
+    #[display(fmt = "Block gas allowance exceeded")]
```

Comment:
Im wondering If we could abstract it since it more about how system implemented rather than each particular builtin